packages/scandipwa/src/route/CategoryPage/CategoryPage.component.js

```diff
@@ -206,13 +206,14 @@ export class CategoryPage extends PureComponent {
         );
     }
 
-    renderLayoutButton = (type) => {
+    renderLayoutButton = (type, index) => {
         const { layout } = this.state;
 
         switch (type) {
         case 'grid':
             return (
                 <button
+                  key={ index }
```

Comment:
Indexes are not valid React keys. Using indexes like this is a common antipattern.
Much more correct way of writing this would be to use `type` or even a string literal `Grid` directly.

Reply:
@yeegor switched to `type`